src/element/textWysiwyg.test.tsx

```diff
@@ -1509,4 +1510,33 @@ describe("textWysiwyg", () => {
       expect(text.text).toBe("Excalidraw");
     });
   });
+
+  it("should update labeled arrow bounds", async () => {
```

Comment:
since the version is being updated now, that's what we should test in this test, can you update the test?

Reply:
Oh ok